docs/docs/07-examples/10-using-javascript.md

```diff
@@ -4,7 +4,73 @@ id: using-javascript
 keywords: [example, javascript, extern, typescript, js, ts]
 ---
 
-Calling a Javascript function from Wing requires two steps.
+## Creating inflight function from JavaScript/TypeScript file
+
+Inflight closures are an extremely important Wing functionality. Consider the following simple wing program:
+
+```wing
+// main.w
+bring cloud;
+let bucket = new cloud.Bucket();
+
+bucket.onCreate(inflight (file) => {
+  log(file);
+});
+```
+
+Being able to write inflight wing alongside the preflight code is beautiful, but you may want to write the inflight function in a separate file and *language*. The special `importInflight` function can be used to create an inflight closure from a JavaScript/TypeScript:
```

Comment:
I am not a fan of this new name. What was the motivation to change from `@inflight()`?

Reply:
Me neither but without the @ I couldn't just call it `inflight`, have any other ideas? I was asked to keep the syntax change out of the PR to possibly revisit later.

Reply:
So let's go with `@inflight` (and follow up by fixing all the rest of the builtins)